### PR TITLE
raiden_contracts: change CONTRACTS_VERSION to 0.38.0

### PR DESCRIPTION
**Pull request by istankovic**
### What this PR does
Change the CONTRACTS_VERSION to 0.38.0.

### Why I'm making this PR
So that raiden uses the latest released contract code.

:warning: This PR should not be merged until we are reasonably sure that the newer contracts will not break tests in the `raiden` repository.

### CODE REVIEW

File: raiden_contracts/constants.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from raiden_contracts.utils.type_aliases import ChainID, Locksroot
 
 # The last digit is supposed to be zero always. See `RELEASE.rst`.
-CONTRACTS_VERSION = "0.37.0"
+CONTRACTS_VERSION = "0.38.0"
 ALDERAAN_VERSION = "0.37.0"
```
Comment (palango) [inline raiden_contracts/constants.py:12]: I guess we should add - for completeness sake - a `BESPIN_VERSION` as well.

Comment (istankovic) [reply raiden_contracts/constants.py:12]: Sounds ok, but it should have the same value as ALDERAAN, right?

Comment (palango) [reply raiden_contracts/constants.py:12]: Yes

**Review by karlb — APPROVED** [review]

**Comment by palango** [comment]
@istankovic Can't we just merge this? Before a new release this cannot be pulled into raiden anyways, right?

**Comment by istankovic** [comment]
> @istankovic Can't we just merge this? Before a new release this cannot be pulled into raiden anyways, right?

I'm afraid we can't just pull this before either
- we fix the code in Raiden repo to support linking

or 

- get rid of split code (i.e. requiring linking) in raiden-contracts

**Comment by palango** [comment]
But why? The raiden repo uses a pinned version of the contracts, right? So even if we change something here, this won't have any impact on the raiden repo.

**Comment by istankovic** [comment]
> But why? The raiden repo uses a pinned version of the contracts, right?

Nope.
Worse, the version constant is being used internally to e.g. access contract-related data.

**Comment by palango** [comment]
> > But why? The raiden repo uses a pinned version of the contracts, right?
> 
> Nope.
> Worse, the version constant is being used internally to e.g. access contract-related data.

https://github.com/raiden-network/raiden/blob/develop/requirements/requirements.txt#L224

**Comment by istankovic** [comment]
> > > But why? The raiden repo uses a pinned version of the contracts, right?
> > 
> > 
> > Nope.
> > Worse, the version constant is being used internally to e.g. access contract-related data.
> 
> https://github.com/raiden-network/raiden/blob/develop/requirements/requirements.txt#L224

But that's not really pinned as it should be -- in the `.in` file. It's just the result of what the latest version was at the time of `deps compile`.

File: raiden_contracts/constants.py (hunk 1 above)
Comment (istankovic) [reply raiden_contracts/constants.py:13]: Done.

**Comment by palango** [comment]
Thank you!

**Pull request merged**